drivers/bridge/link.go

```diff
@@ -27,7 +27,6 @@ func newLink(parentIP, childIP string, ports []types.TransportPort, bridge strin
 		ports:    ports,
 		bridge:   bridge,
 	}
-
```

Comment:
Please remove this change when you push again to take care of the test code changes